libs/wingsdk/src/cloud/function.ts

```diff
@@ -100,6 +101,13 @@ export abstract class FunctionBase extends Resource implements IInflightHost {
       external: ["aws-sdk"],
     });
 
+    // the bundled contains line comments with file paths, which are not useful for us, especially
+    // since they may contain system-specific paths. sadly, esbuild doesn't have a way to disable
+    // this, so we simply filter those out from the bundle.
+    const outlines = readFileSync(outfile, "utf-8").split("\n");
+    const isLineComment = (line: string) => !line.startsWith("//");
```

Comment:
Nit
This returns true when line is not a comment. Which can be confusing since since the filter here is meant to "filter for" and not "filter out". Might be more clear with
```suggestion
    const isNotLineComment = (line: string) => !line.startsWith("//");
    writeFileSync(outfile, outlines.filter(isNotLineComment).join("\n"));
```